platform/.rubocop.yml

```
@@ -52,3 +52,36 @@ RSpec/MultipleMemoizedHelpers:
 
 Style/HashSyntax:
   EnforcedShorthandSyntax: never
+
+RSpec/VerifiedDoubleReference:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 15
+
+Layout/LineLength:
+  Max: 140 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)